user_category.py

```diff
@@ -32,16 +41,12 @@ def user_type(user_category_value):
     elif user_category_value <= 36:
         print("Your user category is: Youth")
         return
-    
-    elif user_category_value <= 0:
```

Comment:
Redundant `elif` you can use `if` instead if you have a return in the first condition